Sources/TuistScaffold/TemplatesDirectoryLocator.swift

```diff
@@ -57,7 +57,7 @@ public final class TemplatesDirectoryLocator: TemplatesDirectoryLocating {
                    share/
                        Templates
                 */
-            bundlePath.parentDirectory.appending(try! RelativePath(validating: "share/Templates")),
+            bundlePath.parentDirectory.appending(try! RelativePath(validating: "share")),
```

Comment:
The line below, `path.appending(component: Constants.templatesDirectoryName)`, is adding the `/Templates` component to the path.